src/server/game/Warden/WardenWin.h

```diff
@@ -89,6 +94,9 @@ class WardenWin : public Warden
 
     std::list<uint16> _CurrentChecks;
     std::list<uint16> _PendingChecks;
+
+    std::list<uint16> _QueuedPayloads;
```

Comment:
From what I've read I don't think you need a doubly linked list. You can use std::forward_list.